### ISSUE TITLE
ACTION-IF-NOT-FOUND with BOOST_FIND_LIB

### ISSUE THREAD

**Issue by gmarcais** [tsuna/boost.m4#76]
I using BOOST_FIND_LIB to check for the availability of a library (regex in my case), but it is not an error for me if it is not present. Simply some feature of the software are not available. In the case where the library is not present, I simply want to set HAVE_BOOST_REGEX to false with AM_CONDITIONAL.

In other word, I am looking for something like this:

```
BOOST_CHECK_LIB([regex]) # set have_boost_regex to yes or no
AM_CONDITIONAL([HAVE_BOOST_REGEX], [test x"$have_boost_regex" = xyes])
```

Is there something equivalent with boost.m4?

**Comment by nlyan** [comment]
This is also a major pain point for me. PowerDNS really needs to be able to test for the existance of  the Boost Context library and switch to a fallback if it's not found. Distros often ship the headers but not the library, if it can't be built on a particular platform.

**Comment by pieterlexis** [comment]
Implemented as a "fail on not found" parameter in #115
